modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/ViewOrganizationsManagementToolbarDisplayContext.java

```diff
@@ -148,32 +150,18 @@ public String getOrderByType() {
 	}
 
 	public PortletURL getPortletURL() {
-		PortletURL portletURL = _renderResponse.createRenderURL();
-
-		portletURL.setParameter("cur", String.valueOf(_getCur()));
-		portletURL.setParameter("delta", String.valueOf(_getDelta()));
-		portletURL.setParameter("displayStyle", _displayStyle);
-
-		String[] keywords = ParamUtil.getStringValues(_request, "keywords");
-
-		if (ArrayUtil.isNotEmpty(keywords)) {
-			portletURL.setParameter("keywords", keywords[keywords.length - 1]);
+		try {
+			return PortletURLUtil.clone(
+				PortletURLUtil.getCurrent(_renderRequest, _renderResponse),
```

Comment:
Since we're cloning the current url every time, we can safely create the current url object in the constructor and make it a final field.  That way we only call `PortletURLUtil.getCurrent` once when the class is created.

The same applies to the other class.